### DISCUSSION TITLE
Not getting location data & fonts sometimes too big

### DISCUSSION THREAD

**Discussion by furtive-one** [tillsteinbach/VWsFriend#362] [General]
Hi
I've just got software v3.0 installed on my ID.3 and am now getting odometer data in VWsFriend, but I am not getting location data, so the start and end locations for trips and journeys are still blank.

I have enabled location sharing in the car, and the location data is being fed ok to Tronity so I know it works. Any idea why it would not be populated in VWsFriend?

Also on some dashboards, the font size is waaay to big in places, for example on the journey dashboard the font in the details pane is size 25 so all th characters are bunched up and overlap. If I edit it and change the font so something like 18 it looks much better, but I can't save the changes.

<img width="273" alt="image" src="https://user-images.githubusercontent.com/93925122/202143621-7691550d-1458-4796-872a-3f9b94af46c7.png">

**Comment by tillsteinbach** [comment]
Regarding the location Problem: Did you restart the container once the update was installed? VWsFriend checks the capabilities of the car only after startup.

regarding font sizes: I have to look. I thought I set everything to „auto“. Maybe the algorithm to determine the size is not good enough. I will have a look.

**Reply by furtive-one** [reply]
Many thanks Till. I had not restarted the container, so have just done so and that will hopefully fix the provlem.

Looking at all the dashbords I think that is the only one with the font issue. All the others seem to be ok
